clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/confirm_process.py

```diff
@@ -0,0 +1,24 @@
+
+# Copyright (C) Metaswitch Networks.
+"""Script to ask the user to confirm a process"""
+
+
+def Confirm_yn(prompt):
+    """asks the user to confirm they want to make the changes described by the
+    prompt passed in. This keeps asking the user until a valid response is
+    given."""
```

Comment:
It's probably also worth mentioning what this function returns (eg. how someone calling it should interpret the returned value).